### ISSUE TITLE
Cannot define empty class

### ISSUE THREAD

**Issue by cstjean** [JuliaPy/PyCall.jl#389]
That's a marginal use case, but still

```julia
@pydef type Empty
end
> BoundsError: attempt to access 0-element Array{Any,1} at index [1]
```

**Comment by stevengj** [comment]
Maybe due to https://github.com/JuliaPy/PyCall.jl/blob/e8f22910ad2f86e2b17333ddafad9ea2ce67da62/src/pyclass.jl#L59-L64 ?

**Comment by cstjean** [comment]
Fixed by #392
